README.md

```diff
@@ -1 +1,411 @@
+
+### Upbound Cloud Hosted UXP Control Plane
+
+Hosted Control planes are run on Upbound's cloud infrastructure and provide a restricted
+Kubernetes API endpoint that can be accessed via `kubectl` or CI/CD systems.
+
+#### Create a free account in Upbound Cloud
+
+1. Sign up for [Upbound Cloud](https://cloud.upbound.io/register).
+1. When you first create an Upbound Account, you can create an Organization
+
+#### Create a Hosted UXP Control Plane in Upbound Cloud
+
+1. Create a `Control Plane` in Upbound Cloud (e.g. dev, staging, or prod).
+1. Connect `kubectl` to your `Control Plane` instance.
+   * Click on your Control Plane
+   * Select the *Connect Using CLI*
+   * Paste the commands to configure your local `kubectl` context
+   * Test your connectivity by running `kubectl get pods -n upbound-system`
+
+#### Installing UXP on a Kubernetes Cluster
+
+The other option is installing UXP into a Kubernetes cluster you manage using `up`, which
+is the official CLI for interacting with Upbound Cloud and Universal Crossplane (UXP).
+
+There are multiple ways to [install up](https://cloud.upbound.io/docs/cli/#install-script),
+including Homebrew and Linux packages.
+
+```console
+curl -sL https://cli.upbound.io | sh
+```
+
+Ensure that your kubectl context is pointing to the correct cluster:
+
+```console
+kubectl config current-context
+```
+
+Install UXP into the `upbound-system` namespace:
+
+```console
+up uxp install
+```
+
+Validate the install using the following command:
+
+```console
+kubectl get all -n upbound-system
+```
+
+#### Install the Crossplane kubectl extension (for convenience)
+
+Now that your kubectl context is configured to connect to a UXP Control Plane,
+we can install this reference platform as a Crossplane package.
+
+```console
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | sh
+cp kubectl-crossplane /usr/local/bin
+```
+
+#### Install the Platform Configuration
+
+```console
+# Check the latest version available in https://cloud.upbound.io/registry/upbound/platform-ref-azure
+PLATFORM_VERSION=v0.2.0
+PLATFORM_CONFIG=registry.upbound.io/upbound/platform-ref-azure:${PLATFORM_VERSION}
+
+kubectl crossplane install configuration ${PLATFORM_CONFIG}
+kubectl get pkg
+```
+
+#### Configure Providers in your Platform
+
+A `ProviderConfig` is used to configure Cloud Provider API credentials. Multiple
+`ProviderConfig`s can be created, each one pointing to a different credential.
+
+In order to manage resources in Azure, you must provide credentials for an Azure service principal that Crossplane can use to authenticate. This assumes that you have already set up the Azure CLI client with your credentials.
+
+Create a JSON file that contains all the information needed to connect and authenticate to Azure:
+
+```console
+# Create service principal with Owner role
+az ad sp create-for-rbac --sdk-auth --role Owner > crossplane-azure-provider-key.json
```

Comment:
Along with a couple of other things to resolve, this is one we should probably address across all of the platform-ref guides. If we agree, I can update all three with this suggestion.

Reply:
Yes i like this suggestion to more clearly identify the account that was created for the context of using this reference platform - we should do that in general across the other platforms.  can you open an issue to track that @natereid72? if we don't include this change in this PR (it would be nice, but not blocking), then let's do them all together in the issue that you'll open :)

Reply:
I've update azure platform ref pr with this suggestion. Will leave this thread open until I confer with @stevendborrelli and create new issue.

Reply:
Will mark as resolved for now. Will follow up with Steven on opening issues in other repos.